Assets/Scripts/Enemy/EnemySpawn.cs

```diff
@@ -193,6 +193,12 @@ void SpawnEnemy(EnemyType enemy)
             variant = EnemyVariantType.NONE;
             enemyType = enemy;
         }
+
+        random = Random.Range(0, 10);
+        if (enemyType == EnemyType.TURRET && random < 5)
+        {
+            variant = EnemyVariantType.SET;
+        }
```

Comment:
not a nitpick but at somepoint we might have to do something about this probability spawning